File: PremiumCalculator/Models/Member.cs
using System.ComponentModel.DataAnnotations;

namespace PremiumCalculator.Models
{
    public class Member
    {
        [Required(ErrorMessage = "Name is required")]
        public string name { get; set; }
        [Required(ErrorMessage = "Age is required")]
        public int age { get; set; }
        [Required(ErrorMessage = "Date of birth is required")]
        public string date_of_birth { get; set; }
        [Required(ErrorMessage = "Death Sum insured is required")]
        public int death_sum_insured { get; set; }
        public Occupation occupation { get; set; }
    }
}

File: PremiumCalculator/ClientApp/src/app/models/member.ts
export interface Member {
    name: string;
    age: number;
    date_of_birth: Date;
    death_sum_insured: number;
    occupation: Occupation;
}

export interface Occupation {
    id: number;
    occupation_desc: string;
    rating: string;
    rating_factor: number;
}